api/src/main/java/com/alibaba/nacos/api/config/ConfigService.java

```diff
@@ -118,6 +118,19 @@ String getConfigAndSignListener(String dataId, String group, long timeoutMs, Lis
     boolean publishConfigCas(String dataId, String group, String content, String casMd5, String type)
             throws NacosException;
     
+    /**
+     * Publish config.
+     *
+     * @param dataId  dataId
+     * @param group   group
+     * @param content content
+     * @param type  type
+     * @return Whether publish
+     * @throws NacosException NacosException
+     * @since 1.4
```

Comment:
this interface has added in 1.4?

Reply:
Sorry, I found that this problem has been fixed in [issue #3907], so I don't need to modify it. Please refuse.

Reply:
Because I found this problem when I used Nacos last August. At that time, this problem had not been solved. For some reasons, I delayed until now to upload my solution.

Reply:
Sorry, this problem has been solved and does not need to be modified. Please refuse.
Because I found this problem when I used Nacos last August. At that time, this problem had not been solved. For some reasons, I delayed until now to upload my solution.

Reply:
Besides, I'm kidding. We're all Chinese. Why not communicate in Chinese？Isn't it better for foreigners to learn Chinese

Reply:
Besides, we are all Chinese. Why not communicate in Chinese? Isn't it better for foreigners to learn Chinese? It's just a joke,

Reply:
Besides,  We're all Chinese. Why not communicate in Chinese？Isn't it better for foreigners to learn Chinese,just kidding.

Reply:
We recommend using English to allow more non-Chinese users to understand the context, but our community does not mandate it.